src/cobalt.rs

```diff
@@ -2,13 +2,16 @@ use std::collections::HashMap;
 use std::fs;
 use std::io::Write;
 use std::path;
+use std::time::SystemTime;
 
 use failure::ResultExt;
 use jsonfeed::Feed;
 use log::debug;
 use log::trace;
 use log::warn;
 use sitemap::writer::SiteMapWriter;
+use vimwiki::vendor::chrono::DateTime;
+use vimwiki::vendor::chrono::Utc;
```

Comment:
We shouldn't be relying on vimwiki like this.  If we need chrono, we should use chrono.